org-formation/600-access/_tasks.yaml

```diff
@@ -1,4 +1,20 @@
 
+JumpcloudiAtlasAdmin:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/IAM/jumpcloud-idp.yaml
+  DependsOn: CommunityIamSamlProviderRP
+  StackName: jc-access-iatlas-admin
+  MaxConcurrentStacks: 10
+  Parameters:
+    MetadataDocument: !ReadFile ./MetadataDocuments/iatlas-admin.xml
+    SamlProviderName: iatlas-admin
+    SamlProviderPolicyArns:
+      - arn:aws:iam::aws:policy/AdministratorAccess
```

Comment:
We'll need to think about if Admin access should be IT-only or if the system owner, like Andrew, should have it... Does is make sense to have a PowerUser role in these accounts that are dedicated to a specific project?

Reply:
yes it does @ahayden.  You are welcome to setup additional JC SSOs by creating similar PRs.